universal-login-web3/src/services/ProviderOnlyIframeInitializer.ts

```diff
@@ -9,9 +9,13 @@ import {SdkConfig} from '@unilogin/sdk';
 export class ProviderOnlyIframeInitializer extends IframeInitializerBase {
   private readonly provider: ULWeb3Provider;
 
-  constructor(endpoint: IframeBridgeEndpoint, network: Network, sdkConfig?: Partial<SdkConfig>) {
+  constructor(
+    endpoint: IframeBridgeEndpoint,
+    network: Network,
+    overrides: {sdkConfig?: Partial<SdkConfig>, showWaitingForTransaction?: boolean},
```

Comment:
* overrides should be optional
* extract {sdkConfig?: Partial<SdkConfig>, showWaitingForTransaction?: boolean} to